packages/components/menu/src/menu-item.vue

```diff
@@ -95,9 +95,11 @@ export default defineComponent({
 
     const handleClick = () => {
       if (!props.disabled) {
-        rootMenu.handleMenuItemClick(
-          reactive({ index: props.index, indexPath, route: props.route })
-        )
```

Comment:
I think we need to double confirm with @Liiked here for the previous detailed informations. Arbitrarily removing this part might broke the component.